packages/pieces/community/pdf-to-text/src/lib/actions/pdf-to-text.ts

```diff
@@ -0,0 +1,24 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { readPdfText } from 'pdf-text-reader';
+
+export const parseTextFromPdfFile = createAction({
+  // auth: check https://www.activepieces.com/docs/developers/piece-reference/authentication,
+  name: 'parseTextFromPdfFile',
+  displayName: 'Parse text from PDF file',
+  description: 'Parses texts from PDF file',
+  props: {
+    file: Property.LongText({
```

Comment:
Let use Property.File

It can parse multiple format (url / base64 / files passed by other steps) into buffer 

https://www.activepieces.com/docs/developers/piece-reference/properties#file

Reply:
Separated the current action into two pieces that parses from PDF file and PDF url